src/v3/test/integration/__snapshots__/authenticator-verification-okta-verify-push-code.test.tsx.snap

```diff
@@ -241,7 +241,36 @@ exports[`authenticator-verification-okta-verify-push-code Polling should display
                     class="MuiBox-root emotion-36"
                     data-se="icon-code"
                   >
-                    <mocksvgicon />
+                    <svg
+                      aria-hidden="true"
+                      aria-labelledby="phoneIcon"
+                      fill="none"
+                      height="63"
+                      role="img"
+                      viewBox="0 0 51 63"
+                      width="51"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <title
+                        id="phoneIcon"
+                      >
+                        L10N_ERROR[icon.title.mobileDevice]
```

Comment:
@glenfannin-okta if a localization key hasn't been translated yet, does it not default to english?